cmd/xmidt-agent/config.go

```diff
@@ -64,9 +64,17 @@ type QOS struct {
 	MaxQueueBytes int64
 	// MaxMessageBytes is the largest allowable wrp message payload.
 	MaxMessageBytes int
-	// Priority determines what is used [newest, oldest message] for QualityOfService tie breakers,
+	// Priority determines what is used [newest, oldest message] for QualityOfService tie breakers and trimming,
 	// with the default being to prioritize the newest messages.
 	Priority qos.PriorityType
+	// LowQOSExpires determines when low qos messages are trimmed.
+	LowQOSExpires time.Duration
```

Comment:
You don't need 'QOS' in the names of these settings, as that's already in the name of the enclosing struct

Reply:
sounds good!